packages/external-db-authorization/package.json

```diff
@@ -25,11 +25,13 @@
   },
   "dependencies": {
     "@aws-sdk/client-cognito-identity-provider": "^3.47.0",
+    "express": "^4.17.2",
```

Comment:
we should not import express here, i thought the security model should include passport here, but not express

Reply:
We decided that the authentication will be middleware, therefore I have to import Express and use it in the packge ..

Reply:
middleware is related to express therefor should be on the main module, the api's being called in it should be on the authorization module
(and change the module to security on another PR)